articles/quantum-InstallConfig.md

```diff
@@ -15,131 +15,28 @@ ms.topic: article
 # Installing the Quantum Development Kit
 
 The Quantum Development Kit can be used either with the Visual Studio 2017 integrated development environment, or with the command line and a development editor such as Visual Studio Code.
-We'll walk through both methods in this install guide.
+
+<table>
+<tr>
+<td>
+[<img src="media/install-guide-select-vs-2017.png" width="90%">](xref:microsoft.quantum.install.vs-2017)
+<div class="nextstepaction">
+[Install for Visual Studio 2017 on Windows 10](xref:microsoft.quantum.install.vs-2017)
+</div>
+</td>
+
+<td>
+[<img src="media/install-guide-select-vs-code.png" width="90%">](xref:microsoft.quantum.install.vs-code)
+<div class="nextstepaction">
+[Install for Visual Studio Code on Windows 10, macOS, or Linux](xref:microsoft.quantum.install.vs-code)
```

Comment:
I would rename this to "Install from the command line on Windows 10, macOS or Linux"

Reply:
Why exactly are we so eager about not having VS Code mentioned in a prominent place? I think it should at least have equal visibility as VS.